gammapy/datasets/spectrum.py

```diff
@@ -477,10 +477,20 @@ def npred_background(self):
             n_on=self.counts.data,
             n_off=self.counts_off.data,
             alpha=self.alpha.data,
-            mu_sig=self.npred_sig().data,
+            mu_sig=self.npred_signal().data,
         )
         return RegionNDMap.from_geom(geom=self._geom, data=mu_bkg)
 
+    def npred_off(self):
+        """Predicted counts in the off region
```

Comment:
Maybe add this is `alpha*npred_background` and link to the Wstat formula?

Reply:
Thanks, indeed. There are probably a few more places where such a link would be useful, I'll check it consistently...

Reply:
Done